### ISSUE TITLE
Copy failed with: fopen () failed

### ISSUE THREAD

**Issue by wtmkleve** [madMAx43v3r/chia-plotter#862]
Hello, since 2 days I have this issue

On other plotters(computers) it do work

I have the newest version 0.1.2a

I already restarted the computer, tried another plot directories and other SSD's

I can't get rid of this issue

I'm using
-n -1 -t E:\temp\ -2 E:\temp (tried also temp2) -d E:\final (tried also other drives!) -r 16 -u 512 -v 128

I don't know what to do anymore

I also changed the permission from all folders to full Access

I'm using windows 10

What I've just see that the plot file has a .plot.tmp extension and the plotter tries to copy the .plot file but it's a .plot.tmp file ?!

**Comment by CleronX** [comment]
you must put a \ at the end of each directory eg. -d E:\final\ (-n 1 -r 16 -u 512 -v 128 -t E:\temp\ -2 E:\temp\ -d E:\final\ )

**Comment by CleronX** [comment]
madmax renames the file from plot.temp to .plot after copy. u can still use the temp file by manualy copying it and renameing the extension

**Comment by wtmkleve** [comment]
> you must put a \ at the end of each directory eg. -d E:\final\ (-n 1 -r 16 -u 512 -v 128 -t E:\temp\ -2 E:\temp\ -d E:\final\ )

Yes, I did that, that was not the issue

I solved it meanwhile

It was my backup software(aomei backupper) for my company Data, that software blocked mmplotter the access to that file

**Comment by wtmkleve** [comment]
> madmax renames the file from plot.temp to .plot after copy. u can still use the temp file by manualy copying it and renameing the extension

Yeah of course, but I can't 24/7 run every 90 minutes to the computer and move and rename the files 🤣

So if anyone else have issues like that, check the backup Software if you have one!!

Or in some cases it could be the antivirus or wrong permissions on that folder

**Comment by programsalon** [comment]
I don't think of it's because of the backup program. I didn't found the error file in ssd tmp driver and any other hd drivers, it lost!
